assets/js/blocks/active-filters/block.json

```diff
@@ -1,17 +1,19 @@
 {
 	"name": "woocommerce/active-filters",
 	"version": "1.0.0",
-	"title": "Active Product Filters",
+	"title": "Active Product Filter Controls",
 	"description": "Display the currently active product filters.",
 	"category": "woocommerce",
 	"keywords": [ "WooCommerce" ],
 	"supports": {
 		"html": false,
 		"multiple": false,
+		"inserter": false,
 		"color": {
 			"text": true,
 			"background": false
-		}
+		},
+		"lock": false
```

Comment:
Just to confirm my assumption: We set the lock status to `false` initially for users who already have the block inserted, but once they upgrade to the new inner blocks version we set this to true so users cannot remove the inner blocks?

Reply:
### With `lock` support disabled:

<img width="1317" alt="image" src="https://user-images.githubusercontent.com/5423135/194275608-532f26f0-2e1e-4785-ad88-fcf0d6863d49.png">

### With `lock` support enabled:

<img width="1288" alt="image" src="https://user-images.githubusercontent.com/5423135/194275875-0f8c9373-3bf1-4b68-97b2-3be769960f14.png">